### PR TITLE
Updated French translation

### PR DESCRIPTION
**Pull request by SombrAbsol**
I just finished translating the strings on RetroAchievements.

One question though, where is the "Play" button file? I haven't found it and would like to translate it as well.
Screenshot_20230228_224759_melonDS

### CODE REVIEW

**Comment by rafaelvcaetano** [comment]
You can find it at the top of the list of strings. Line 19.

`<string name="play">Play</string>`

Since it's a generic string, I like to put them at the top.

File: app/src/main/res/values-b+fr/strings.xml [hunk 1]
```diff
@@ -288,6 +290,23 @@
     <string name="background_add_wrong_orientation">L\'arrière-plan a été ajouté mais n\'est pas visible pour l\'orientation sélectionnée.</string>
     <string name="background_add_processing_failed">Le traitement de l\'image d\'arrière-plan sélectionnée a échoué. Vérifiez qu\'elle n\'est pas corrompue.</string>
 
+    <string name="retro_achievements_login_description">Connectez-vous à RetroAchievements pour consulter les succès que vous pouvez déverouiller en jouant à ce jeu !</string>
```
Comment (rafaelvcaetano) [inline app/src/main/res/values-b+fr/strings.xml:293]: Be careful with the extra space before the final exclamation mark! (Unless this is the way it's done in French).

File: app/src/main/res/values-b+fr/strings.xml [hunk 2]
```diff
@@ -288,6 +290,23 @@
...
+    <string name="username">Nom d\'utilisateur</string>
+    <string name="password">Mot de passe</string>
+    <string name="retry">Réessayer</string>
+    <string name="points_abbreviated">PTS</string>
+    <string name="points">Points</string>
+    <string name="completed">Terminé</string>
+    <string name="view_achievement">Consulter le succès</string>
+    <string name="completed_achievements"><b>%1$d</b> sur <b>%2$d</b> (%3$d %%)</string>
+    <string name="submitting_achievements">Envoi des RetroAchievements…</string>
+    <string name="achievement_unlocked">Succès déverouillé !</string>
```
Comment (rafaelvcaetano) [inline app/src/main/res/values-b+fr/strings.xml:308]: Extra space before the exclamation mark also present here.

**Comment by SombrAbsol** [comment]
For the space before the !, don't worry, it's normal. In French, there is a space before some punctuation marks (! ? : ; % for example). And thanks for the answer to my question 👍

**Comment by SombrAbsol** [comment]
Achievement unlocked!
"French translation" - 10 PTS

**Comment by rafaelvcaetano** [comment]
Sorry about leaving this here for so long! I didn't want to give the work of constantly opening new PRs after every new string I added. But I think that all text changes for the next release are done. Or at least, I will try to avoid changing anything in that regard. If you still want to update the strings, I think now is the perfect time! And thanks for your effort!

**Comment by SombrAbsol** [comment]
No problem, I don't mind at all.👌
I'll check the strings one last time later.

**Comment by SombrAbsol** [comment]
It should be done now :)

**Pull request merged**